accepted/2021/package-validation/package-validation.md

```diff
@@ -0,0 +1,190 @@
+# Package Validation
+
+**Owner** [Immo Landwerth](https://github.com/terrajobst)
+
+With .NET Core & Xamarin we have made cross-platform a mainstream requirement
+for library authors. However, we lack validation to ensure that library
+developers, which can result in packages that don't work well which in turn
```

Comment:
"to ensure that library developers" do what exactly?

Reply:
BE BETTER THAN LAST TIME 😄